tests/js/valueWithId.tests.ts

```diff
@@ -0,0 +1,96 @@
+import { expect } from "chai";
+
+import { RoundingMethodEnum } from "../../src/js/math";
+import { RoundedValueWithId, ValueWithId } from "../../src/js/ValueWithId";
+
+const defaultId = 0;
+const defaultValue = null;
+
+const testId = 1;
+const testValue = "testValue";
+const differentValue = "differentValue";
```

Comment:
In Py tests we defined the values at the top with human-readable names, maybe we can do it here too